src/integrators/integrator_utils.jl

```diff
@@ -61,10 +61,11 @@ function _savevalues!(integrator, force_save, reduce_size)::Tuple{Bool, Bool}
     saved, savedexactly = false, false
     !integrator.opts.save_on && return saved, savedexactly
     tdir_t = integrator.tdir * integrator.t
-    while !isempty(integrator.opts.saveat) && first(integrator.opts.saveat) <= tdir_t # Perform saveat
+    saveat = integrator.opts.saveat
+    while !isempty(saveat) && first(saveat) <= tdir_t # Perform saveat
         integrator.saveiter += 1
         saved = true
-        curt = integrator.tdir * pop!(integrator.opts.saveat)
+        curt = integrator.tdir * pop!(saveat)
```

Comment:
I assume this doesn't make a major difference?

Reply:
Yeah, I just did that while looking at where all the copies came from.
I thought, maybe it's thinking that stuff possibly gets mutated on each iter and is thus reloading on each call.
But I didn't notice any difference.